build_runner/bin/graph_inspector.dart

```diff
@@ -0,0 +1,203 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+import 'dart:async';
```

Comment:
[nit] I'm also a fan of a blank line under the copyright before the first import

Reply:
done